safe-global/web-core
# feat: relay transaction execution modal

## What it solves

Partially resolves https://github.com/safe-global/web-core/issues/1700

## How this PR fixes it
Implements the Figma designs. The `AdvancedParams` accordion will be addressed in a different PR.
Manages a `executionMethod` state var in the `SignOrExecuteForm` component to style the modal accordingly.
Fetches from the relay service the remaining hourly relays.

## How to test it
1. Create a tx that can be immediately executed.
2. If the "Execute transaction" is checked, a radio group to chose the execution method will be visible.
3. The modal should behave like illustrated in Figma (except the AdvancedParams) https://www.figma.com/file/M3J3IJ5YFxRKaT95F2uBAt/Relaying?node-id=1250%3A31675&t=kF4YHMdwVG5UZbzS-0

## Screenshots
![Screenshot 2023-03-01 at 17 05 45](https://user-images.githubusercontent.com/32431609/222195445-a3b1f64c-1d97-44d0-8293-992798795504.png)

## Checklist
